Makefile

```diff
@@ -626,6 +626,12 @@ install-completions:
 clean:
 	$(QUIET_CLEAN)rm -f $(TARGET) $(SHIMV2) $(NETMON_TARGET) $(CONFIGS) $(GENERATED_FILES) .git-commit .git-commit.tmp
 
+vendor:
+	export GO111MODULE=on \
+	go mod tidy && \
+	go mod vendor && \
+	go mod verify
```

Comment:
Nice. Could you add a check like this to https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh for repos that are using go modules? Yes, I know the check is already here, but having such a central and PR-gating check will hopefully keep all the repos protected.

Reply:
@jodh-intel I have finally addressed this after my unexpected interim break.
I have opened a PR in the tests repo to account for go modules.
See: https://github.com/kata-containers/tests/pull/1880

Reply:
Thanks @amshinde.